deprecated/class-yoast-modal.php

```diff
@@ -0,0 +1,79 @@
+<?php
+/**
+ * @package WPSEO\Admin
+ */
+
+/**
+ * Class to implement a React modal.
+ *
+ * @deprecated 9.2
+ */
+class Yoast_Modal {
+
+	/** @var array The modal configuration. */
+	private static $config = array();
+
+	/**
+	 * Class constructor.
+	 *
+	 * @deprecated 9.2
+	 */
+	public function __construct() {
+		_deprecated_constructor( 'Yoast_Modal', '9.2' );
+	}
+
+	/**
+	 * Enqueues the assets needed for the modal.
+	 *
+	 * @deprecated 9.2
+	 *
+	 * @return void
+	 */
+	public function enqueue_assets() {
+		_deprecated_function( 'Yoast_Modal::enqueue_assets', '9.2' );
```

Comment:
`'Yoast_Modal::enqueue_assets'` could be replaced by '__METHOD__', like in the other deprecated files. This would be more consistent with the other files. Same goes of every deprecated function in this file.

Reply:
We not using the `__METHOD__`  constant everywhere.  I think it's not a big deal and this make it more readable.